File: client/web/sudoo/lib/data/api/auth/request/change_password_request.dart
import 'package:sudoo/data/api/base_request.dart';

class ChangePasswordRequest extends BaseRequest{
  final String oldPassword;
  final String newPassword;

  ChangePasswordRequest({required this.oldPassword, required this.newPassword});

  @override
  Map<String, dynamic> toJson() {
    return {
      "oldPassword": oldPassword,
      "newPassword": newPassword,
    };
  }
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/api/auth/request/ChangePasswordRequest.kt
package com.sudo248.sudoo.data.api.auth.request


/**
 * **Created by**
 *
 * @author *Sudo248*
 * @since 09:37 - 04/03/2023
 */
data class ChangePasswordRequest(
    val oldPassword: String,
    val newPassword: String,
)